### DISCUSSION TITLE
ebusd & Esera

### DISCUSSION THREAD

**Discussion by gerd2005** [john30/ebusd#1407] [Q&A]
Guten Tag,

funktioniert ebusd in der Kombination mit dem Gateway von Esera (https://esera.de/Produkte/Heizungsoptimierung-eBus-und-Heizungsbus-Systeme/eBus-Gateway-300er-Serie/)?

Benötige ich dafür noch weitere Hardware o.ä.?

Danke und viele Grüße
gerd

**Comment by e1l52** [comment]
Hi Gerd,

ich versuche es gerade ebenfalls mit einem ECO305. Bin aber noch nicht erfolgreich.
Siehe hier : https://github.com/john30/ebusd/discussions/1411

Gruß Reinhold
